<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>FDFZer's 垃圾系统</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="labelHistory">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>80</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>历史记录：</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelPoint">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>30</y>
      <width>72</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>积分</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelPointHint">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>30</y>
      <width>72</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>积分：</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidgetHistory">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>120</y>
      <width>481</width>
      <height>431</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelUserIdHint">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>30</y>
      <width>72</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>用户ID：</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelUserId">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>30</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>用户ID</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelHint">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>40</y>
      <width>251</width>
      <height>361</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>叶根友唐楷简</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>等待用户中...</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonUserLogin">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>460</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>模拟用户识别</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonThrow">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>510</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>模拟丢垃圾</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelTimerHint">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>410</y>
      <width>72</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>倒计时：</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelTimer">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>410</y>
      <width>72</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>倒计时</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSystem">
    <property name="title">
     <string>系统</string>
    </property>
    <addaction name="actionAdminLogin"/>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuSystem"/>
  </widget>
  <action name="actionAdminLogin">
   <property name="text">
    <string>管理员登录</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>退出系统</string>
   </property>
  </action>
  <action name="actionHistory">
   <property name="text">
    <string>历史记录</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
